src/Microsoft.OData.Edm/Csdl/Serialization/EdmModelCsdlSchemaXmlWriter.cs

```diff
@@ -525,7 +508,7 @@ internal override void WriteVocabularyAnnotationElementHeader(IEdmVocabularyAnno
             this.WriteRequiredAttribute(CsdlConstants.Attribute_Term, annotation.Term, this.TermAsXml);
             this.WriteOptionalAttribute(CsdlConstants.Attribute_Qualifier, annotation.Qualifier, EdmValueWriter.StringAsXml);
 
-            if (isInline && !IsUsingDefaultValue(annotation))
+            if (isInline && !annotation.UseDefault)

```

Comment:
Will this retain the same behavior as the deleted function?

Reply:
Yes